src/webview/js/rewriteHtml.js

```diff
@@ -56,4 +55,46 @@ const rewriteImageUrls = (auth: Auth, element: Element | Document = document) =>
   });
 };
 
-export default rewriteImageUrls;
+/* Rewrite the 'time' elements under the given root, inclusive, to make them more readable.
+ *
+ * 1. Changes text to localized human readable date-time.
+ * 2. Adds 'original-text' attribrute, to show original text alert dialog on press.
+ */
+const rewriteTime = (element: Element | Document) => {
+  // Find the image elements to act on.
+  const timeElements = [].concat(
+    element instanceof HTMLTimeElement ? [element] : [],
+    Array.from(element.getElementsByTagName('time')),
+  );
+
+  timeElements.forEach(elem => {
+    const timeStamp = elem.getAttribute('datetime');
+    const text = elem.innerText;
+
+    // 'text' and 'timeStamp' are guaranteed to be present. The conditional is
+    // present only to satisfy Flow.
```

Comment:
This kind of comment makes me ask: how do you know / why are they guaranteed? Or put another way: what do you know that Flow doesn't? That's the question I most want answered from a comment here.

There's a similar conditional above in this file which provides a nice example of answering that question:
```
    // Not `img.src`, because that getter absolutizes relative URLs itself.
    const actualSrc = img.getAttribute('src');

    // A missing `src` is invalid per the HTML spec.
    if (actualSrc == null) {
      return;
    }
```

OK, Flow doesn't know that fact. It *does* actually know that an `HTMLImageElement` must have a `src: string` -- but we avoided `img.src` for the reason mentioned in the other comment, and instead wrote `img.getAttribute('src')`, and Flow can't make the connection between the string passed to `getAttribute` and the type.

But it's not obvious to me that there's a good excuse like that here.